pkg/processor/runtime/dotnetcore/Wrapper.cs

```diff
@@ -80,7 +139,7 @@ private void MessageReceived(object sender, EventArgs e)
             {
                 var st = new System.Diagnostics.Stopwatch();
                 Response response = null;
-                var context = new Context();
```

Comment:
since you have moved the context variable initializtion up, you may need to remove below 
`context.Logger.LogEvent += LogEvent;`
and
`context.Logger.LogEvent -= LogEvent;`

Reply:
Are you referring to removing these two lines in the `MessageReceived()` method?

Reply:
yes